client/landing/jetpack-cloud/components/sidebar/index.jsx

```diff
@@ -48,9 +48,11 @@ class JetpackCloudSidebar extends Component {
 		const { selectedSiteSlug, translate } = this.props;
 
 		return (
-			<Sidebar>
-				<CurrentSite />
+			<Sidebar
+				className="is-jetpack-cloud-sidebar" // eslint-disable-line wpcalypso/jsx-classname-namespace
```

Comment:
What if we use use `className="sidebar sidebar--jetpack-cloud"`? The `eslint` comment can be deleted and `"sidebar--jetpack-cloud"` better conveys this is a modifier of the original sidebar.

Reply:
:+1: